test/js/pageobjects/CrossDeviceEverythingWeNeed.js

```diff
@@ -0,0 +1,28 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class CrossDeviceEverythingWeNeed extends Base {
+
+  get crossDeviceEverythingWeNeedTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
```

Comment:
I don't think you need to use crossDevice in front of each method name as the class will already tell you that you are testing cross device

Reply:
Done.